src/src/PluginStructs/P134_data_struct.cpp

```diff
@@ -0,0 +1,122 @@
+#include "../PluginStructs/P134_data_struct.h"
+
+#ifdef USES_P134
+
+# ifndef LIMIT_BUILD_SIZE
+const __FlashStringHelper* toString(A02YYUW_status_e status) {
+  switch (status) {
+    case A02YYUW_status_e::STATUS_OK: return F("OK");
+    case A02YYUW_status_e::STATUS_ERROR_CHECK_SUM: return F("Checksum error");
+    case A02YYUW_status_e::STATUS_ERROR_MAX_LIMIT: return F("Distance over max. limit");
+    case A02YYUW_status_e::STATUS_ERROR_MIN_LIMIT: return F("Distance under min. limit");
+    case A02YYUW_status_e::STATUS_ERROR_SERIAL: return F("Serial communication error");
+  }
+  return F("Unknown");
+}
+
+# endif // ifndef LIMIT_BUILD_SIZE
+
+/**************************************************************************
+* Constructor
+**************************************************************************/
+P134_data_struct::P134_data_struct(uint8_t config_port,
+                                   int8_t  config_pin1,
+                                   int8_t  config_pin2)
+  : _config_port(config_port), _config_pin1(config_pin1), _config_pin2(config_pin2)
+{
+  const ESPEasySerialPort port = static_cast<ESPEasySerialPort>(_config_port);
+
+  delete P134_Serial;
```

Comment:
Why is delete needed in the constructor? The member should be made nullptr initially instead. There simply cannot be an object assigned to the pointer during the constructor.

Reply:
I CC'ed that from the original code. Should have removed it.

Reply:
> Should have removed it.

Yep you should ;)